pkg/server/scheme/scheme.go

```diff
@@ -59,4 +61,80 @@ var (
 
 	// ParameterCodec handles versioning of objects that are converted to query parameters.
 	ParameterCodec = runtime.NewParameterCodec(Scheme)
+
+	// GenericControlPlaneGroupName is the name of the group when installed in the generic control plane
+	GenericControlPlaneGroupName = "core"
+
+	// GenericControlPlaneSchemeGroupVersion is group version used to register these objects
+	GenericControlPlaneSchemeGroupVersion = schema.GroupVersion{Group: GenericControlPlaneGroupName, Version: "v1"}
 )
+
+// AddKnownTypes adds the set of types defined in the Kubernetes API to the Scheme.
+// This is minimal set of types that are required to be registered in the scheme in order to
+// serve KCP control plane usecases.
+func AddKnownTypes(scheme *runtime.Scheme) error {
```

Comment:
No idea if this wrong/right

Reply:
don't like it. Why should we have our own set of types in a scheme? Why not just call the `Install` func for core?

Reply:
Me too. TBH was experimenting to make it running as it was not for me. 
Do we have `Install` function already?